### PR TITLE
Fix some broken quotes in the SMBLoris section

### PR DESCRIPTION
**Pull request by rbsec**
The quote mark around SMBLoris was broken in two of the documents, and showed an unprintable character.

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

File: windows/whats-new/ltsc/whats-new-windows-10-2019.md [hunk 1]
```diff
@@ -220,7 +220,7 @@ The new [security baseline for Windows 10 version 1803](https://docs.microsoft.c
 
 #### SMBLoris vulnerability
 
-An issue, known as “SMBLoris�?, which could result in denial of service, has been addressed.
+An issue, known as “SMBLoris”, which could result in denial of service, has been addressed.
```
Comment (illfated) [inline windows/whats-new/ltsc/whats-new-windows-10-2019.md:223]: Thank you. There is another issue present: MS Docs prefers to use ASCII quote characters (as do I), and let the docs.microsoft.com CSS handle the typeface fonts:
```suggestion
An issue, known as "SMBLoris", which could result in denial of service, has been addressed.
```

File: windows/whats-new/whats-new-windows-10-version-1709.md [hunk 2]
```diff
@@ -119,7 +119,7 @@ The minimum PIN length is being changed from 6 to 4, with a default of 6. For mo
 Microsoft has released new [Windows security baselines](https://docs.microsoft.com/windows/device-security/windows-security-baselines) for Windows Server and Windows 10. A security baseline is a group of Microsoft-recommended configuration settings with an explanation of their security impact. For more information, and to download the Policy Analyzer tool, see [Microsoft Security Compliance Toolkit 1.0](https://docs.microsoft.com/windows/device-security/security-compliance-toolkit-10).
 
 ### SMBLoris vulnerability
-An issue, known as “SMBLoris�?, which could result in denial of service, has been addressed.
+An issue, known as “SMBLoris”, which could result in denial of service, has been addressed.
```
Comment (illfated) [inline windows/whats-new/whats-new-windows-10-version-1709.md:122]: (same as above - ASCII vs ANSI characters)
```suggestion
An issue, known as "SMBLoris", which could result in denial of service, has been addressed.
```

File: windows/whats-new/ltsc/whats-new-windows-10-2019.md (hunk 1 above)
Comment (rbsec) [reply windows/whats-new/ltsc/whats-new-windows-10-2019.md]: Hi @illfated,

I wasn't sure what the convention was for these documents - there was  mixture of different quote types in this one. I've updated these (and also a couple of other instances in the same file) to ASCII quotes in dc686fc .

File: windows/whats-new/whats-new-windows-10-version-1709.md (hunk 2 above)
Comment (rbsec) [reply windows/whats-new/whats-new-windows-10-version-1709.md]: Fixed in c0f41ea1fb.

**Review by illfated — APPROVED** [review]
Thank you for these updates. Looks good to me.

**Comment by illfated** [comment]
#sign-off

File: windows/whats-new/ltsc/whats-new-windows-10-2019.md (hunk 1 above)
Comment (illfated) [reply windows/whats-new/ltsc/whats-new-windows-10-2019.md]: Very good job. Thank you, well done. 🙂

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**